js/ZeroMusic.coffee

```diff
@@ -109,6 +116,7 @@ class ZeroMusic extends ZeroFrame
       return @cmd "wrapperNotification", ["error", "Only mp3 files are allowed for now.", 5000]
     @cmd "dbQuery", ["SELECT MAX(id) + 1 as next_id FROM songs"], (res) =>
       id = if res[0] then res[0].next_id else 1
+      if id == null then id = 1;
```

Comment:
I think you could merge the line you added in the previous one, with something like this:
```coffee
id = if res[0] and res[0].next_id then res[0].next_id else 1
```